### ISSUE TITLE
[BUG] DHCP Ethernet Fails Connect with Firmware 6.1.3 (See Also Issue 4160)

### ISSUE THREAD

**Issue by edtyler** [prusa3d/Prusa-Firmware-Buddy#4233]
### Printer model

XL

### Firmware version

6.1.3

### Printing from...

None

### Describe the bug

Ethernet Does not accept address offered by DHCP server:Oct 02 19:35:21 snow.tylerent.com dhcpd[1093402]: DHCPDISCOVER from 10:9c:70:22:24:be via enp0s25
Oct 02 19:35:21 snow.tylerent.com dhcpd[1093402]: DHCPOFFER on 10.48.96.52 to 10:9c:70:22:24:be via enp0s25

The XL does not respond to the DHCPOFFER.

I am unable to use wireless for security reasons.

No other packets are being sent by the printer:
tcpdump -vv -i enp0s25 ether host 10:9c:70:22:24:be
dropped privs to tcpdump
tcpdump: listening on enp0s25, link-type EN10MB (Ethernet), snapshot length 262144 bytes
19:58:07.378512 IP (tos 0x0, ttl 255, id 2, offset 0, flags [none], proto UDP (17), length 336)
    0.0.0.0.bootpc > 255.255.255.255.bootps: [udp sum ok] BOOTP/DHCP, Request from 10:9c:70:22:24:be (oui Unknown), length 308, xid 0x21625adf, Flags [none] (0x0000)
	  Client-Ethernet-Address 10:9c:70:22:24:be (oui Unknown)
	  Vendor-rfc1048 Extensions
	    Magic Cookie 0x63825363
	    DHCP-Message (53), length 1: Discover
	    MSZ (57), length 2: 1500
	    Parameter-Request (55), length 4: 
	      Subnet-Mask (1), Default-Gateway (3), BR (28), Domain-Name-Server (6)
19:58:07.378650 IP (tos 0x10, ttl 128, id 0, offset 0, flags [none], proto UDP (17), length 328)
    snow.tylerent.com.bootps > 10.48.96.52.bootpc: [udp sum ok] BOOTP/DHCP, Reply, length 300, xid 0x21625adf, Flags [none] (0x0000)
	  Your-IP 10.48.96.52
	  Client-Ethernet-Address 10:9c:70:22:24:be (oui Unknown)
	  Vendor-rfc1048 Extensions
	    Magic Cookie 0x63825363
	    DHCP-Message (53), length 1: Offer
	    Server-ID (54), length 4: snow.tylerent.com
	    Lease-Time (51), length 4: 3600
	    Subnet-Mask (1), length 4: 255.255.255.0
	    Default-Gateway (3), length 4: snow.tylerent.com
	    BR (28), length 4: 10.48.96.255
	    Domain-Name-Server (6), length 4: snow.tylerent.com
19:58:11.378499 IP (tos 0x0, ttl 255, id 4, offset 0, flags [none], proto UDP (17), length 336)
    0.0.0.0.bootpc > 255.255.255.255.bootps: [udp sum ok] BOOTP/DHCP, Request from 10:9c:70:22:24:be (oui Unknown), length 308, xid 0x21625adf, Flags [none] (0x0000)
	  Client-Ethernet-Address 10:9c:70:22:24:be (oui Unknown)
	  Vendor-rfc1048 Extensions
	    Magic Cookie 0x63825363
	    DHCP-Message (53), length 1: Discover
	    MSZ (57), length 2: 1500
	    Parameter-Request (55), length 4: 
	      Subnet-Mask (1), Default-Gateway (3), BR (28), Domain-Name-Server (6)
19:58:11.391610 IP (tos 0x10, ttl 128, id 0, offset 0, flags [none], proto UDP (17), length 328)
    snow.tylerent.com.bootps > 10.48.96.52.bootpc: [udp sum ok] BOOTP/DHCP, Reply, length 300, xid 0x21625adf, Flags [none] (0x0000)
	  Your-IP 10.48.96.52
	  Client-Ethernet-Address 10:9c:70:22:24:be (oui Unknown)
	  Vendor-rfc1048 Extensions
	    Magic Cookie 0x63825363
	    DHCP-Message (53), length 1: Offer
	    Server-ID (54), length 4: snow.tylerent.com
	    Lease-Time (51), length 4: 3600
	    Subnet-Mask (1), length 4: 255.255.255.0
	    Default-Gateway (3), length 4: snow.tylerent.com
	    BR (28), length 4: 10.48.96.255
	    Domain-Name-Server (6), length 4: snow.tylerent.com

### How to reproduce

1) Configure DHCP on printer (default)
2) Power on printer,  plug in Ethernet cable, observe lights on connector are flashing
3) No IP address is assigned

### Expected behavior

I would expect the printer to accept the DHCPOFFER and begin using the assigned IP address.

**Comment by edtyler** [comment]
Thanks for helping me understand the settings interface. I mounted a USB stick on a Linux box and created the .ini file with these settings:
[eth::ipv4]
type=static
addr=172.16.10.71
mask=255.255.255.0
gw=172.16.10.253

[network]
dns4=10.48.96.37

Next, I used Settings > System > Load Settings from File. The printer loaded the settings, but indicated the connection failed. Both ping of gateway and dns showed 100% packet loss. 

At this point, I started to suspect something else was going on. I brought out my NetTool Pro and observed that the link was going up and down at about a 5 second interval. 

Long story short, the house cable was installed a long time ago and even though it was CAT5, it was not properly pinned for EIA-568A/B. When I configured the link for 10-BaseT, it worked fine. Re terminating both ends of the house cable got the speeds to 1000-BaseT.

Bottom line, the printer works fine, will accept either DHCP or  a statically configured address. This was completely the fault of the house wiring.

Please close this case as USER ERROR. I will also request close of #4234.
